autoextract_poet/page_inputs.py

```diff
@@ -38,34 +38,44 @@ class _AutoExtractData(Generic[T]):
 
     https://doc.scrapinghub.com/autoextract.html#responses
     """
-
-    item_key: ClassVar[str]
-
+    page_type: ClassVar[str]
     data: dict
+    _item: Optional[T] = attr.ib(init=False, repr=False)
+
+    def __attrs_post_init__(self):
+        # Caching item to avoid building it again and again
+        self._item = self.item_class.from_dict(self.data[self.page_type])
```

Comment:
This may also lead to unnecessary creation of item - to_item overhead is paid even if an user decided not to call it for some reason. Though I may be missing the reason for adding a cache here.

Reply:
Well, it is very unlikely that the user won't use the `to_item()` method, so I don't think is an overhead. But maybe there is no strong reason to have a cache as you tell.

Reply:
This method was removed in https://github.com/scrapinghub/autoextract-poet/pull/9/commits/c5111d906ba544cf4680a6e33bd6c9c09825cc80